tests/Feature/Http/Controllers/Auth/EmailVerificationTest.php

```diff
@@ -48,7 +48,7 @@ public function testEmailVerificationScreenIsNotRenderedWhenEmailIsAlreadyVerifi
         $this->markTestSkipped();
 
         // Given
-        $user = User::factory()->create([
+        $user = AuthorizedUser::factory()->create([
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create([
````